#Represents a named point.

# General info.
Header header

# Name.
string name

# 3D point.
geometry_msgs/Point point
#geometry_msgs/Pose pose
